### ISSUE TITLE
Will this work with srcset/picture?

### ISSUE THREAD

**Issue by iamdubx** [matheusgrieger/vue-clazy-load#6]
Hi. Is there an option to support responsive images?

**Comment by matheusgrieger** [comment]
Hello! No, currently there isn't, but it's perfectly doable. I'll add it to the 1.0 roadmap.
Thanks for the request!

**Comment by liborvanek** [comment]
Hi, any plan to implement this?

**Comment by matheusgrieger** [comment]
Yes! It's going to be implemented in v1.0. I'm just busy with some other stuff and haven't got the time to develop it until now, but it's in the works! :smile:

**Comment by matheusgrieger** [comment]
Hello @equinusocio! Yes, this will be supported, following the `picture` tag guidelines and docs. Sadly, I had no time to work on it at the moment, as I'm currently with my head fully occupied by other projects.

I hope that I will be coming back to Clazy Load soon, though. Sorry for the delay on this issue.
